nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -1091,9 +1103,12 @@ in
       nix-store = mkIf cfg.mountHostNixStore {
         source = builtins.storeDir;
         target = "/nix/store";
+        securityModel = "none";
       };
       xchg = {
         source = ''"$TMPDIR"/xchg'';
+        securityModel = "none";
+        securityModel = "none";
```

Comment:
seems like it was added twice here, instead of to `virtualisation.sharedDirectories.shared` and `virtualisation.sharedDirectories.certs`